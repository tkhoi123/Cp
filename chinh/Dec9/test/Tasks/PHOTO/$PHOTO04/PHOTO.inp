8
7 5 6 2 3 8 4 1 
7 5 6 1 3 8 2 4 
5 3 6 8 2 4 7 1 
7 6 3 5 8 2 4 1 
7 5 8 6 3 2 4 1 
