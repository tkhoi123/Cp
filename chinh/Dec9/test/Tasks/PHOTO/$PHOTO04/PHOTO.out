7 5 6 3 8 2 4 1 